src/renderer/components/storm/StormPlan.tsx

```diff
@@ -268,27 +269,35 @@ const StormPlanEditorInner = withPlannerContext(
             );
         }
 
-        const hovered = useAtomValue(atomHoveredChatUI);
-        switch (hovered.type) {
+        const clicked = useAtomValue(atomClickedChatUI);
+        const setClicked = useSetAtom(atomClicked);
```

Comment:
I think this is the one-liner way to do it
```suggestion
        const [clicked, setClicked] = useAtom(atomClicked);
```